Source/options.cpp

```diff
@@ -157,6 +157,15 @@ void SaveIni()
 	IniChanged = false;
 }
 
+#if SDL_VERSION_ATLEAST(2, 0, 0)
+bool HardwareCursorDefault()
+{
+	SDL_version v;
+	SDL_GetVersion(&v);
+	return SDL_VERSIONNUM(v.major, v.minor, v.patch) >= SDL_VERSIONNUM(2, 0, 12);

```

Comment:
Maybe I'm just being silly, but a safer comparison would be:
```suggestion
	return v.major > 1 && (v.major > 2 || v.minor > 0 || v.patch >= 12);
```

Reply:
I think we can trust the SDL-provided macro. That's what SDL uses anyway for things like `SDL_VERSION_ATLEAST`.

Reply:
That's at least a comfort :)